HTML/mockup-minimo/style.css

```diff
@@ -0,0 +1,360 @@
+        margin-top: 8rem;
+        grid-template: 1fr 1fr 1fr 1fr / 1fr;
+        grid-column-gap: 0rem;
+        grid-row-gap: 7rem;
+    }
+}
+
+.article__header {
+    margin-bottom: 3.4rem;
+    font-family: PlayfairDisplay;
+    font-size: 3rem;
+    text-align: left;
+    color: #626262;
+}
+
+@media only screen and (max-width: 800px) {
+    .article__header {
+        margin-bottom: 1.5rem;
+        font-size: 2.8rem;
+    }
+}
+
+.article__tag {
+    margin-bottom: 2.8rem;
+    text-transform: uppercase;
+    font-family: Ubuntu;
+    font-size: 1.4rem;
+    font-weight: 300;
+    text-align: left;
+    color: #b4ad9e;
+}
+
+@media only screen and (max-width: 800px) {
+    .article__tag {
+        margin-bottom: 1.6rem;
+    }
+}
+
+.article__tag:last-child {
+    margin-top: 5.2rem;
+    margin-bottom: 0;
+}
+
+.article__paragraph {
+    padding-right: 0.6rem;
+    font-family: PlayfairDisplay;
+    font-size: 1.4rem;
+    line-height: 1.71;
+    text-align: left;
+    color: #626262;
+}
+
+picture {
+    display: block;
+}
+
+.article__image {
+    margin-bottom: 3.2rem;
+    width: 100%;
+}
+
+/****** SECTION_SIGN_IN ******/
+
+.section-sign-in {
+    margin-top: 12.2rem;
+    padding: 7.3rem 41.3rem;
+    width: 100%;
+    background-color: #f0f0f0;
+}
+
+@media only screen and (max-width: 1200px) {
+    .section-sign-in {
+        padding: 7.3rem 20%;
+    }
+}
+
+@media only screen and (max-width: 800px) {
+    .section-sign-in {
+        padding: 7rem 15%;
+    }
+}
+
+@media only screen and (max-width: 500px) {
+    .section-sign-in {
+        padding: 5.3rem 10%;
+    }
+}
+
+.section-sign-in__header.article__header {
+    margin-bottom: 8.3rem;
+    letter-spacing: 0.8px;
+    color: #333333;
+}
+
+@media only screen and (max-width: 800px) {
+    .section-sign-in__header.article__header {
+        margin-bottom: 4.3rem;
+    }
+}
+
+.section-sign-in__form {
+    padding-bottom: 1.4rem;
+    display: flex;
+    justify-content: space-between;
+    border-bottom: solid 1px #000000;
+}
+
+input {
+    border: none;
+    background-color: transparent;
+    box-shadow: none;
+}
+
+.section-sign-in__input {
+    width: 90%;
+}
+
+.section-sign-in__input::placeholder {
+    font-family: PlayfairDisplay;
+    font-size: 1.4rem;
+    line-height: 1.71;
+    text-align: left;
+    color: #626262;
+}
+
+.section-sign-in__button {
+    font-size: 2.3rem;
+}
+
+/****** SECTION_POSTS_SECOND ******/
+
+.section-posts--second {
+    margin-top: 10.3rem;
+    display: grid;
+    grid-template: 1fr / 1fr 1fr;
+    grid-column-gap: 6rem;
+}
+
+@media only screen and (max-width: 800px) {
+    .section-posts--second {
+        grid-template: 1fr 1fr / 1fr;
+        grid-row-gap: 7rem;
+    }
+}
+
+.section-posts__button {
+    margin-top: 16.5rem;
+    text-align: center;
+}
+
+@media only screen and (max-width: 800px) {
+    .section-posts__button {
+        margin-top: 10rem;
+    }
+}
+
+.button {
+    padding: 2.1rem 0;
+    width: 18.2%;
```

Comment:
let's button be content-sizable
fixed width:
![image](https://user-images.githubusercontent.com/11594423/59272104-8c920780-8c55-11e9-82c6-98136a79e64e.png)
content sizeable: 
![image](https://user-images.githubusercontent.com/11594423/59272139-a16e9b00-8c55-11e9-9ce8-4eeaac2c0271.png)